spec/requests/edit_user_spec.rb

```diff
@@ -116,27 +113,12 @@ def sign_in_as_a_valid_user(user)
     end
   end
 
-  context 'user adds a new mobile number but not mobile 2FA' do
-    before do
-      sign_in_as_a_valid_user(user)
-      patch_via_redirect '/users', id: @user, user: attrs_with_email_2fa
-    end
-
-    it 'deletes unconfirmed number after log out if not confirmed' do
```

Comment:
Yes, it was a duplicate. All of the specs in this file are related to this PR, which changes the code related to updating a user, so I think it makes sense to make any changes to this file in this PR. I'm not sure it's worth opening a separate PR just to remove a duplicate spec. To me, a separate PR would make sense for changes that are not related to this PR.